spec/helpers/date_helper_spec.rb

```diff
@@ -21,5 +21,10 @@
       params = {day: "40", month: "13", year: "2020"}
       expect(helper.format_date(params)).to eq(nil)
     end
+
+    it "returns nil when given a zero parameter" do
```

Comment:
The commit message confused me for a while. I was expecting some code in DateHelper but there is only this extra test?

Reply:
Good point, I'll amend the commit message